Jenkinsfile

```diff
@@ -96,7 +96,7 @@ pipeline {
                                                 credentials('brave-builds-github-token-for-pr-builder')
                                                 github('brave/devops', 'https')
                                             }
-                                            branch('master')
+                                            branch('mplesa-jenkins-goma')
```

Comment:
I'm not familiar with this file so I'm not sure if this change was intentional or a leftover from local experimentation. Feel free to ignore if it's correct.